charts/lagoon-logging/ci/linter-values.yaml

```diff
@@ -44,6 +44,7 @@ tls:
     FejDrD4asORcLF9xN2tsM5fGoBuHy9KpUw==
     -----END EC PRIVATE KEY-----
 
+fluentdreplicaCount: 1
```

Comment:
I just realised the capitalisation is not correct here so you'll still get the default `3`.